### PR TITLE
Add toString() for DML event

### PR DESCRIPTION
**Pull request by yjhawar**

### CODE REVIEW

File: delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java [hunk 1]
```diff
@@ -121,6 +122,22 @@ public boolean equals(Object o) {
       Objects.equals(sortKeys, dmlEvent.sortKeys);
   }
 
+  @Override
+  public String toString() {
+    StringJoiner stringJoiner = new StringJoiner(", ", "{", "}");
+    stringJoiner.add("operation=" + operation);
+    stringJoiner.add("isSnapshot=" + this.isSnapshot());
+    stringJoiner.add("changeType=" + this.getChangeType());
+    stringJoiner.add("offset=" + this.getOffset());
```
Comment (sumitjnn) [inline delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java:131]: Is offset guaranteed to be not null ?

File: delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java [hunk 2]
```diff
@@ -121,6 +122,22 @@ public boolean equals(Object o) {
...
+    if (this.getSourceTimestampMillis() != null) {
+      stringJoiner.add("sourceTimestampMillis=" + this.getSourceTimestampMillis());
+    }
+    if (sortKeys != null && !sortKeys.isEmpty()) {
```
Comment (sumitjnn) [inline delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java:135]: nit: we can use Collections.isEmpty instead

**Review by sumitjnn — APPROVED** [review]
Just few minor comments

File: delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java (hunk 1 above)
Comment (yjhawar) [reply delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java:131]: offset can be null in that case it returns an empty map.

File: delta-api/pom.xml [hunk 3]
```diff
@@ -42,6 +42,12 @@
       <version>6.7.0-SNAPSHOT</version>
       <scope>compile</scope>
     </dependency>
+      <dependency>
+          <groupId>commons-collections</groupId>
+          <artifactId>commons-collections</artifactId>
```
Comment (sumitjnn) [inline delta-api/pom.xml]: My bad, I thought we already had commons collection, let's not add a dependency just for one empty check

Comment (yjhawar) [reply delta-api/pom.xml]: However CollectionUtils is a basic library which is generally used right ,so can always have basic use cases

Comment (sumitjnn) [reply delta-api/pom.xml]: Yes it's a basic library but we should have good reason to add a dependency as it adds maintenance overhead like upgrades due to security vulnerabilities, I don't think replacing few null checks justifies it

**Review by sumitjnn — APPROVED** [review]

**Pull request merged**